src/services/infinispanRefData.ts

```diff
@@ -226,6 +226,6 @@ export enum CounterType {
 }
 
 export enum CounterStorage {
-  VOLATILE = 'VOLATILE',
-  PERSISTENT = 'PERSISTENT'
+  VOLATILE = 'Volatile',
```

Comment:
@dvagnero We need to change filtering function if we change from all-caps to Capitalization in CounterStorage.

Do you recommend not using all-caps?

Reply:
Sorry, I didn't know. On my end it changed capitalization for the counter filter and not inside the table. 
![image](https://user-images.githubusercontent.com/58083949/205039056-6a02797f-e76e-4feb-a075-8a34283e93b2.png)
Is there any reason for capitalization in this case? What are you trying to achieve? 
In general, capitalize the abbreviations and optionally some error/warning (like WARNING: Do not ....) . Capitalizing every letter in other cases is perceived as screaming at the user :) 
If you want to highlight the info you might want to change the style, make it bold

Reply:
Actually, the variable from backend we are receiving is in all-caps.
So that's why I was previously using VOLATILE/PERSISTENT.

Now if you check with the recent changes filtering with counter storage is not working. We need to change filtering function.

Reply:
Anyways, I also prefer your way.

Making necessary changes :+1:

Reply:
You're right filtering based on counter storage doesn't work

Reply:
> You're right filtering based on counter storage doesn't work

I've added new commit, now it's working

Reply:
@dvagnero are you making any other changes?

Reply:
I'm done with the changes. From docs perspective it looks good